containers/runtime/xds/config/README.md

```diff
@@ -0,0 +1,33 @@
+# Configuration for xDS server
+
+Configuration JSON files are used to generate the resource snapshot for xDS server. The xDS server consumes these configuration and server these configuration to xDS client or Envoy sidecar.
```

Comment:
I'd suggest "xDS configuration" instead of "resource snapshot". "Resource snapshot" is sort of an internal name to `go-control-plane`.